draft-js-plugins-editor/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - `decorators` don't decorate plugins anymore.
 
+### Changed
+
+- *Breaking change* The `plugins` and `decorators` editor props need to be an Immutable.List()
```

Comment:
editorState is of course also a complex object. I choose to use Immutable.List for both plugins and decorators as it makes comparing them in the componentWillReceiveProps a lot more fault tolerant. And since this check happens on every props change I also wanted to optimise for that. Do you have a different suggestion on how to solve that part?